src/api/app/controllers/webui/request_controller.rb

```diff
@@ -401,4 +378,18 @@ def forward_request_to(fwd)
     request_link = ActionController::Base.helpers.link_to(forwarded_request.number, request_show_path(forwarded_request.number))
     flash[:notice] += " and forwarded to #{target_link} (request #{request_link})"
   end
+
+  def request_action_attributes(type)
+    {
+      target_project:    params[:project],
+      target_package:    params[:package],
+      source_project:    params[:devel_project],
+      source_package:    params[:devel_package] || params[:package],
+      target_repository: params[:repository],
+      person_name:       params[:user],
+      group_name:        params[:group],
+      role:              params[:role],
+      type:              type.to_s
```

Comment:
why don't you pass the type already as string? :thinking:

Reply:
I am lazy:laughing:. And this way it's more generic.